src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -243,19 +234,46 @@ private static String formatLocalDateTime(LocalDateTime localDateTime, String pa
     }
 
     /**
-     * Formats a date in the format dd/MM/yyyy.
+     * Format a datetime stamp from a {@code localDateTime}.
+     * Example: Tue, 12 Apr 2018, 12:01 PM
```

Comment:
An example that shows a leading zero (e.g. 01 Apr) is better as it shows how single-digit date values are formatted. Same for all the other javadoc.